packages/react/src/components/Checkbox/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+const Checkbox = ({
+  isUndefined,
+  disabled,
+  children,
+  isChecked: isDefaultChecked,
+  sublabel,
+  onChange,
+}) => {
+  const [isChecked, setIsChecked] = useState(isDefaultChecked);
+
+  const onCheckboxStateChange = e => {
+    const { checked } = e.target;
+    setIsChecked(checked);
+    onChange && onChange(e);
+  };
+
+  const buildInputTag = () => (
+    <input
+      checked={isChecked}
+      disabled={disabled}
+      className={cx("Checkbox", { "is-undefined": isUndefined })}
+      type="checkbox"
+      onChange={onCheckboxStateChange}
+    />
+  );
+
+  return children ? (
+    <label className="Checkbox-label">
+      {buildInputTag()}
+      <div>
+        {children}
+        {sublabel && <p className="Checkbox-sublabel">{sublabel}</p>}
+      </div>
+    </label>
+  ) : (
+    buildInputTag()
+  );
+};
+
+Checkbox.propTypes = {
+  onChange: PropTypes.func,
+  isUndefined: PropTypes.bool,
+  isChecked: PropTypes.bool,
+  disabled: PropTypes.bool,
+  children: PropTypes.string,
```

Comment:
Why would this not be required?
Also, should be `PropTypes.node`

Reply:
* Remove condition for labels
* ESlint - check config in Pemo's VSCode
* Children not required - ask Juan
* Different content for children - ask Juan
* remove label-has-for

Reply:
I've left conditions for leaving the labels, because if `Checkbox` was left in `label` without an actual label, the object looked like:

![image](https://user-images.githubusercontent.com/15340486/62120432-e1e7ae00-b2c1-11e9-87e0-aa959df72a10.png)

Rest Juan covered in Slack thread